src/components/SignupForm.js

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
```

Comment:
I thought about combining them but then I would have to work a lot with conditional rendering I guess. I wasn't sure if the outcome will be looking better or worse :D

Reply:
I guess the only difference is the auth method and some text or am I wrong?

Reply:
Yes, the auth method and the signup form has one more input field for the password confirmation. Probably I will add another one for the name in the future. And yes, here and there some text (for example on the button) has to change.

Reply:
ok then its enough to justify a separate component. But you could outsource all the styled components, group them in the LoginForm folder and import those in both the LoginForm and Signup Form. I had the same issue in my capstone project and it was a real struggle to change the styling since I had to change the same components at three different places...

Reply:
I did it now as you suggested and I guess it is better that way. I could get rid of 3 files at least and the form doesn't look too messy. Have a look!
On top of that, I think soon I will do some refactoring and will then also extract the styled-components as you suggested. As I use them for the login/signup form but also for the forget-password-form